app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -180,6 +187,11 @@
                Can't be greater than {{limits.max | usageWithUnits : 'storage'}}.
              </span>
            </div>
+           <div ng-if="persistentVolumeClaimForm.capacity.$error.outOfStorage" class="has-error">
```

Comment:
given the context this validity is used in, it is strange that my PVC quota being exceeded would flag an error on this field.  i would expect you to only be checking requests.storage here